homeassistant/components/bluetooth/common_device.py

```diff
@@ -0,0 +1,24 @@
+"""Common functions related to Bluetooth device management."""
```

Comment:
We try to avoid having members called via sub-modules.
It's ok to have a sub-module for this, but I think it should be imported and re-exported in `__init__.py` so it can be imported with:
```python
from homeassistant.components.bluetooth import (
    sensor_device_info_to_device_info,
)
```

Reply:
On second thought, would this be better in `homeassistant.helpers.bluetooth`..?